<launch>

  <arg name="goal_input" default="231"/>
  <arg name="goal_distance" default="40"/>

  <group>
    <node pkg="my_package" type="localization_node_test.py" name="localization_node_test" output="screen" args="$(arg goal_input) $(arg goal_distance)">
      <param name="goal_input" type="int" value="$(arg goal_input)" />
      <param name="goal_distance" type="int" value="$(arg goal_distance)" />
    </node>
    <node pkg="my_package" type="state_estimation.py" name="state_estimation" output="screen"/>
  </group>

</launch>